node-data/src/message.rs

```diff
@@ -1242,7 +1242,10 @@ impl StepMessage for Candidate {
         &mut self.sign_info
     }
     fn signable(&self) -> Vec<u8> {
-        self.candidate.header().hash.to_vec()
+        let mut signable = self.header.signable();
```

Comment:
```suggestion
        let mut signable = self.header.signable();
        signable.extend_from_slice(Self::SIGN_SEED);
```

I would like to add the SIGN_SEED too (I know it's empty, but in case we change it we don't struggle it adapt this code too